tests/RobotFramework/tests/cumulocity/software.robot

```diff
@@ -16,6 +16,21 @@ Install software via Cumulocity
     Operation Should Be SUCCESSFUL           ${OPERATION}    timeout=60
     Device Should Have Installed Software    c8y-remote-access-plugin
 
+# TODO: make it not flaky
+tedge-agent should terminate on SIGINT while downloading file
+    # we download a file which is 500M, but tmpfs at /tmp is only 64M
```

Comment:
So what are the impacts on the test? Do you want to test how the system behaves in that case? Or do you try to SIGINT the download before?

Reply:
I meant to explain why we change tmp.path config setting. Will correct comment to explain intent.